### PR TITLE
Run Models from File

### PR DESCRIPTION
**Pull request by BlairArchibald**
These commits streamline the creation of models, including letting them run from parameters specified in files.

### CODE REVIEW

File: src/main/java/uk/co/ramp/covid/simulation/Model.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+            }
+            if (schoolLockDown != null) {
+                p.setSchoolLockdown(schoolLockDown.start, schoolLockDown.end, schoolLockDown.socialDistance);
+            }
+            stats.add(p.timeStep(nDays));
+        }
+
+        if (outputFile != null) {
+            for (int i = 0; i < nIters; i++) {
+                outputCSV(outputFile, i, stats.get(i));
```
Comment (johnnonweiler) [inline src/main/java/uk/co/ramp/covid/simulation/Model.java:175]: There's only one filename, but the outputCSV method is written as though it is writing a new file each time (e.g. creating a new FileWriter, and printing the headers).  What is the intention here?

File: src/main/java/uk/co/ramp/covid/simulation/Model.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            LOGGER.warn("Uninitialised model parameter: nDays");
+            valid = false;
+        }
+        if (outputFile == null) {
+            LOGGER.warn("Uninitialised model parameter: outputFile");
+            valid = false;
+        }
+        // Handle optional args
+        if (lockDown != null) {
+            LOGGER.warn("lockDown parameters invalid");
```
[the remark this answers is not in the document]
Comment (BlairArchibald) [reply src/main/java/uk/co/ramp/covid/simulation/Model.java:142]: Looks like I had this code in here temporarily before I started checking the inner params - great spot! I'll move it now!

File: src/main/java/uk/co/ramp/covid/simulation/Model.java (hunk 1 above)
Comment (BlairArchibald) [reply src/main/java/uk/co/ramp/covid/simulation/Model.java:175]: Again looks like temporary code that's snuck in! I'll fix this up now

**Comment by BlairArchibald** [comment]
These two issues should be fixed now

**Comment by johnnonweiler** [comment]
Looks good.

**Comment by BlairArchibald** [comment]
This is now ready to merge (not sure if you get a notification automatically without me commenting)

**Pull request merged**